spec/factories/message.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :message do
+    text { Faker::Lorem.sentence(word_count: 3) }
+    user
+    association :conversation
```

Comment:
Just `conversation` should work.
You can use `association` when the name is different, for example: `association :another_name, factory: conversation`